docs/Release_notes.md

```diff
@@ -1,6 +1,199 @@
+
+### **CARLA ScenarioRunner** 
+
+The scenario-runner is a new repository and docker image for this release. This directory contains custom ScenarioRunner scenario configurations to facilitate integration testing. This still a work in progress, so the scenarios serve more as example references. ScenarioRunner's code resides in the srunner Python package, and the scenario_runner.py script is responsible for launching scenarios. 
+
+### **EVC-SUMO (New Private Repository)** 
+
+This EVC-sumo is a new private repository. Econolite Virtual Controller (EVC)-SUMO bridge tool that is a part of XiL co-simulation tool. This bridge acts as a mediator between EVC (via the PyEOS Python package) and SUMO, integrating the exchange of traffic light status and detector status.
+Currently, in order to build and use the docker image, the tool needs proprietary github-token accessible to the members of the usdot-fhwa-stol organization due to Econolite's license.
+
+### **Multiple Object Tracking** 
+
+Multiple Object Tracking is a new repository that holds a library to support cooperative perception. This library is intended for tracking multiple objects from multiple sources where the input is abstracted as a standard cooperative perception object tracking interface. For example, object level data can come from multiple sources including J2334 Sensor Data Sharing Message, Basic Safety Message, or local perception. This enables the library to be deployed in different road actors such as C-ADS equipped vehicles or the infrastructure with the help of message adapters that are suited for respective middleware. 
+The library exposes multiple submodules and functionalities adopted from the architectural and algorithmic advancements made by the sensor fusion community. Example usage of this library to fully execute multiple object tracking pipeline is implemented in CARMA Platform here.   
+
+### **CARMA Builds** 
+
+CARMA Builds is a new component of the CARMA ecosystem, which enables a coordination among different transportation users. This component provides Docker images used to build other projects within CARMA. 
+
+### **CARMA Streets** 
+
+Main new functionality in CARMA Streets is that it can now generate J2334 Sensor Data Sharing Message from simulation sensor detected data. It also received further improvements such as a new dependency on CARMA Builds to efficiently pull from common base images and minor improvements on time synchronization, data collection, and json parsing.
```

Comment:
The main new functionality in CARMA Streets is its ability to generate J2334 Sensor Data Sharing Messages from object-level data detected by simulation sensors. Additionally, it has received further enhancements, including a new dependency on CARMA Builds, which allows for efficient pulling from common base images. There are also minor improvements in time synchronization, data collection, and JSON parsing, enhancing the overall functionality and efficiency of the system.

Reply:
updated